### ISSUE TITLE
SDXL training problem

### ISSUE THREAD

**Issue by tasfdtsagda** [bmaltais/kohya_ss#3164]
GUI 2417:
.toml
bucket_reso_steps = 64
cache_latents = true
cache_latents_to_disk = true
caption_extension = ".txt"
clip_skip = 1
dynamo_backend = "no"
enable_bucket = true
epoch = 10
gradient_accumulation_steps = 1
gradient_checkpointing = true
huber_c = 0.1
huber_schedule = "snr"
learning_rate = 0.0005
loss_type = "l2"
lr_scheduler = "cosine_with_restarts"
lr_scheduler_args = []
lr_scheduler_num_cycles = 3
lr_scheduler_power = 1
max_bucket_reso = 2048
max_data_loader_n_workers = 0
max_grad_norm = 1
max_timestep = 1000
max_token_length = 225
max_train_epochs = 0
max_train_steps = 500
min_bucket_reso = 256
min_snr_gamma = 5
mixed_precision = "bf16"
multires_noise_discount = 0.3
multires_noise_iterations = 6
network_alpha = 16
network_args = []
network_dim = 32
network_module = "networks.lora"
no_half_vae = true
noise_offset = 0.1
noise_offset_type = "Original"
optimizer_args = [ "scale_parameter=False", "relative_step=False", "warmup_init=False",]
optimizer_type = "Adafactor"
output_dir = "D:/kohya_models"
output_name = "noobaitest"
pretrained_model_name_or_path = "D:/ComfyUI_windows_portable/ComfyUI/models/checkpoints/SDXL/NoobAI-XL (NAI-XL)/noobaiXLNAIXL_vPred10Version.safetensors"
prior_loss_weight = 1
resolution = "1024,1024"
sample_prompts = "D:/kohya_models\\prompt.txt"
sample_sampler = "euler_a"
save_every_n_epochs = 1
save_model_as = "safetensors"
save_precision = "fp16"
text_encoder_lr = 5e-5
train_batch_size = 4
train_data_dir = "C:/Users/Name/Desktop/test/img"
unet_lr = 0.0005
v_parameterization = true
xformers = true
------------------------
GUI 2503
toml.
bucket_reso_steps = 64
cache_latents = true
cache_latents_to_disk = true
caption_extension = ".txt"
clip_skip = 1
dynamo_backend = "no"
enable_bucket = true
epoch = 10
gradient_accumulation_steps = 1
gradient_checkpointing = true
huber_c = 0.1
huber_scale = 1
huber_schedule = "snr"
loss_type = "l2"
lr_scheduler = "cosine_with_restarts"
lr_scheduler_args = []
lr_scheduler_num_cycles = 3
lr_scheduler_power = 1
max_bucket_reso = 2048
max_data_loader_n_workers = 0
max_grad_norm = 1
max_timestep = 1000
max_token_length = 225
max_train_epochs = 0
max_train_steps = 500
min_bucket_reso = 256
min_snr_gamma = 5
mixed_precision = "bf16"
network_alpha = 16
network_args = []
network_dim = 32
network_module = "networks.lora"
no_half_vae = true
noise_offset = 0.1
noise_offset_type = "Original"
optimizer_args = [ "scale_parameter=False", "relative_step=False", "warmup_init=False",]
optimizer_type = "Adafactor"
output_dir = "D:/kohya_models"
output_name = "noobaitest"
pretrained_model_name_or_path = "D:/ComfyUI_windows_portable/ComfyUI/models/checkpoints/SDXL/NoobAI-XL (NAI-XL)/noobaiXLNAIXL_vPred10Version.safetensors"
prior_loss_weight = 1
resolution = "1024,1024"
sample_prompts = "D:/kohya_models\\sample/prompt.txt"
sample_sampler = "euler_a"
save_every_n_epochs = 1
save_model_as = "safetensors"
save_precision = "fp16"
text_encoder_lr = [ 5e-5, 5e-5,]
train_batch_size = 4
train_data_dir = "C:/Users/Name/Desktop/test/img"
unet_lr = 0.0005
v_parameterization = true
wandb_run_name = "noobaitest"
xformers = true
------------------------
for some reason some values disappear in the new version, others are duplicated.
2417:
text_encoder_lr = 5e-5
learning_rate = 0.0005
multires_noise_discount = 0.3
multires_noise_iterations = 6
2503:
text_encoder_lr = [ 5e-5, 5e-5,]

both were trained using the same configuration.
-------------------------------------------------------------
if i check the raw metada of the lora is the same issue
gui 2417
ss_learning_rate0.0005
ss_multires_noise_iterations6

gui2503

ss_learning_rate2e-06
ss_multires_noise_iterationsNone

**Comment by bmaltais** [comment]
I will need to check why those values are not making it to the toml. Can you share the json config? I will try to reproduce it at my end.

**Comment by tasfdtsagda** [comment]
[noobaiv1.json](https://github.com/user-attachments/files/19612240/noobaiv1.json)

PD: you can't activate SDXL and v_param in the GUI at the same time, you need to modify the .json file to do so.

**Comment by bmaltais** [comment]
OK, the multires issue is that I think the original json does not contain the right noise offset type. It is set to Original in the json and it need to be set to Multires instead for the values to be used. I must have discovered this as a requirement to properly work in the latest sd-scripts and introduced a change to that effect.

So this explain why this one does not work.

Now... let see about the learning rate...

**Comment by bmaltais** [comment]
OK, for the learning rate issue... sd-scripts has added a bunch of new learning rate parameters and the old learning rate one is sort of deprecated... but it you want to match what you had before, set the text encoder learning rate and unet learning rate to 0 and the "legacy" learning rate will be used:

![Image](https://github.com/user-attachments/assets/f3388b62-5333-450b-a85b-95d6067cec0b)

**Comment by bmaltais** [comment]
OK, issue with v_param in gui is now fixed in the dev branch. To test:

```
git checkout dev
git pull
```
